main.js

```diff
@@ -0,0 +1,134 @@
+    }
+    resetGame();
+};
+
+function resetGame() {
+    startButton.disabled = false;
+}
+
+function winConditions() {
+    if (board.get(0) === board.get(1) && board.get(0) === board.get(2) && board.get(0) != '') {
+        win();
+    } else if (board.get(3) === board.get(4) && board.get(3) === board.get(5) && board.get(3) != '') {
+        win();
+    } else if (board.get(6) === board.get(7) && board.get(6) === board.get(8) && board.get(6) != '') {
+        win();
+    } else if (board.get(0) === board.get(3) && board.get(0) === board.get(6) && board.get(0) != '') {
+        win();
+    } else if (board.get(1) === board.get(4) && board.get(1) === board.get(7) && board.get(1) != '') {
+        win();
+    } else if (board.get(2) === board.get(5) && board.get(2) === board.get(8) && board.get(2) != '') {
+        win();
+    } else if (board.get(0) === board.get(4) && board.get(0) === board.get(8) && board.get(0) != '') {
+        win();
+    } else if (board.get(2) === board.get(4) && board.get(2) === board.get(6) && board.get(2) != '') {
+        win();
+    } else if (turnCount === 9) {
+        playerTurn.innerHTML = "It's a TIE!";
+        resetGame();
+    };
+
+};
+startButton.addEventListener('click', disableButton);
+cell0.addEventListener('click', function () { clickCell(0) });
+cell1.addEventListener('click', function () { clickCell(1) });
+cell2.addEventListener('click', function () { clickCell(2) });
+cell3.addEventListener('click', function () { clickCell(3) });
+cell4.addEventListener('click', function () { clickCell(4) });
+cell5.addEventListener('click', function () { clickCell(5) });
+cell6.addEventListener('click', function () { clickCell(6) });
+cell7.addEventListener('click', function () { clickCell(7) });
+cell8.addEventListener('click', function () { clickCell(8) });
+
+function player(){
+    let evenOrOdd = turnCount%2
+    if (evenOrOdd === 0){
+        playerTurn.innerHTML = "Player O's Turn"
+    }else {
+        playerTurn.innerHTML = "Player X's Turn"
+    };
+
+};
+// disables start button first time it is pressed
+function disableButton() {
+    this.disabled = true;
+    resetBoard();
+    playerTurn.innerHTML = 'Player Xs Turn';
+};
+
+function resetBoard() {
+    board = new Map([[0, ''], [1, ''], [2, ''], [3, ''], [4, ''], [5, ''], [6, ''], [7, ''], [8, '']]);
+    turn = true;
+    turnCount = 0;
+    activateBoard();
+    buildBoard();
+};
+
+function activateBoard(){
+    for(let k = 0; k<=8; k++){
+        let id = "cell-" + k;
+        document.getElementById(id).style.pointerEvents = 'auto';
+    }
+};
+
+function clickCell(cell) {
+    console.log("click " + cell);
+    disableCell(cell);
+    if (turn) {
+        player();
+        setCell(cell, 'X');
+        turn = !turn;
+    } else {
+        player();
+        setCell(cell, 'O');
+        turn = !turn;
+    }
+};
+
+function disableCell(cell) {
+    let id = "cell-" + cell;
+    document.getElementById(id).style.pointerEvents = 'none';
```

Comment:
instead of doing a separate DOM query here just pass the cell directly into the function via its variable name when you use it. The proper way to remove an event listener is `target.removeEventListener('event`, eventHandler)` target in this case is the cell the event is a 'click' and our handler is an anonymous function, which does pose some issues, but can be easily remedied by using a named function (`clickCell` will work nicely with some minor tweaks)